### DISCUSSION TITLE
clifm: No data directory found. Data files, such as plugins and color schemes, might not be available. W2green-beret

### DISCUSSION THREAD

**Discussion by LinArcX** [leo-arch/clifm#284] [Q&A]
I'm on windows and built clifm with msys2.

By default clifm working perfectly. the problem happens when i want to set/change the prompt. for example by adding:
`Prompt="green-beret"`
To: 
`~\.config\clifm\colors\default-256.clifm`

i will receive the error i mentioned.

**Comment by LinArcX** [comment]
Good to mention that on msys environment that i have, there was no **share/** directory in **/usr/local**. So i created it by hand. 

After i added this line into my `~/.bashrc`:
`export CLIFM_DATADIR="/usr/local/share/"`

But still i see the same error when i run clifm.

**Comment by leo-arch** [comment]
Have you installed Clifm (via `sudo make install`) or you just compiled and executed the binary?

**Reply by leo-arch** [reply]
Hey @LinArcX, is your problem solved?

**Comment by leo-arch** [comment]
Hi @LinArcX. I don't know if your problem is solved, but forgot to mention that you can use `--data-dir` to specify a custom data directory. Make it point to wherever data files have been installed (make sure to install with `sudo make install` and pay attention to where data files are copied). For example, if they're installed in `/usr/local/share/clifm` write `clifm --data-dir=/usr/local/share` (no ending `clifm`).

This should be the content of **clifm**'s data dir:

```
colors/     plugins/  actions.clifm  keybindings.clifm  nets.clifm     prompts.clifm
functions/  tools/    clifmrc         mimelist.clifm     preview.clifm  readline.clifm
```
To manually build a minimal data directory copy both [clifmrc](https://github.com/leo-arch/clifm/blob/master/misc/clifmrc) and [prompts.clifm](https://github.com/leo-arch/clifm/blob/master/misc/prompts.clifm) to say `~/clifm-data/clifm/`. Then run as follows: `clifm --data-dir ~/clifm-data/`.
